include/translated-object.php

```diff
@@ -165,7 +165,7 @@ protected function should_update_translation_group( $id, $translations ) {
 	public function save_translations( $id, $translations ) {
 		$id = (int) $id;
 
-		if ( ( $lang = $this->get_language( $id ) ) && isset( $translations ) && is_array( $translations ) ) {
```

Comment:
Beware that `is_array()` returns a PHP Notice when passed an undefined variable.

Reply:
I removed the `isset()` because `$translations` is passed to the function and is not optional so is always set.